src/Build/Logging/FancyLogger/FancyLoggerBuffer.cs

```diff
@@ -206,15 +172,10 @@ public static int GetLineIndexById(int lineId)
         }
 
         // Update line
+        // TODO: Remove. Use line.Text instead
         public static FancyLoggerBufferLine? UpdateLine(int lineId, string text)
```

Comment:
@rainersigwald If I understand correctly, removing this might cause breaking changes and the tests to fail, right?

Reply:
I'm not sure I understand this, what is the context?

Reply:
`UpdateLine` was used to update the contents of a line by finding its Id. However I realized this is very slow and it would be more straightforward to just update the Text property of the line. Hence, I would like to delete this method but wasn't sure if doing so would be a breaking change. 
I think it won't matter after we make everything internal.